torch/csrc/jit/passes/quantization.cpp

```diff
@@ -986,10 +988,29 @@ void InsertObserversHelper::preprocess(
   }
 }
 
-// Returns true if the value is the weight to LSTM operator.
-bool isDynamicLSTMWeight(Value* v, Use use, bool is_dynamic) {
-  return is_dynamic && use.user->kind() == Symbol::aten("lstm") &&
-      (use.offset == 2);
+bool useQuantizable(const Use& use, bool is_dynamic) {
+  bool result = nodeQuantizable(use.user);
+  if (!result) {
+    return false;
+  }
+  // Special checks for ops that do not require observers for all input tensors.
+  // For each operator in this list observers are inserted for the input based
+  // on the index specified.
+  const AtenFuncArgs& aten_func_args = AtenFuncArgs({{"lstm", 2}});
```

Comment:
we also need to check `is_dynamic` for "lstm" node..

Reply:
I think that won't be necessary because for dynamic quant we insert observers at the input. Which should be same as static quant. We have a check for `is_dynamic` to insert observer at the output of node in the parent function.

Reply:
OK, then do you need `is_dynamic` as argument here?

Reply:
we don't support static quant right now though